src/restoration-state-handler.ts

```diff
@@ -354,7 +356,31 @@ export class RestorationStateHandler
     return restorationState;
   }
 
-  // remove optional opening and closing quotes from a string
+  /**
+   * Converts a URL sort param into a field/direction pair, if possible.
+   * Either or both may be undefined if the param is not in a recognized format.
+   */
+  private getSortFieldAndDirection(sortParam: string) {
```

Comment:
Sort of -- this parsing code already existed above, in the restoration method. I just pulled it out into its own method for DRY purposes, since the same functionality is needed in the state persistence method now too.